packages/core/src/diff/changes/argument.ts

```diff
@@ -39,12 +40,19 @@ export function fieldArgumentDefaultChanged(
     type: ChangeType.FieldArgumentDefaultChanged,
     message:
       typeof oldArg.defaultValue === 'undefined'
-        ? `Default value '${newArg.defaultValue}' was added to argument '${newArg.name}' on field '${type.name}.${field.name}'`
-        : `Default value for argument '${newArg.name}' on field '${type.name}.${field.name}' changed from '${oldArg.defaultValue}' to '${newArg.defaultValue}'`,
+        ? `Default value '${safeString(newArg.defaultValue)}' was added to argument '${newArg.name}' on field '${type.name}.${field.name}'`
+        : `Default value for argument '${newArg.name}' on field '${type.name}.${field.name}' changed from '${safeString(oldArg.defaultValue)}' to '${safeString(newArg.defaultValue)}'`,
     path: [type.name, field.name, oldArg.name].join('.'),
   };
 }
 
+function safeString(obj: any) {
+  if (typeof Object.getPrototypeOf(obj).toString === 'function') {
```

Comment:
issue: This will still throw an error with objects that have null as the prototype.

```suggestion
if (obj != null && typeof obj.toString === 'function') {
```